javascript/node/selenium-webdriver/test/cookie_test.js

```diff
@@ -168,6 +168,26 @@ suite(function(env) {
       await driver.sleep(expirationDelay);
       await assertHasCookies();
     });
+
+    it('can add same site cookie property to `Strict`', async function() {
+      let cookie = createSameSiteCookieSpec('Strict');
+      let childUrl = fileserver.whereIs('child/childPage.html');
+      await driver.get(childUrl);
+      await driver.manage().addCookie(cookie);
+      await driver.manage().getCookie(cookie.name).then(function(actual) {
+        assert.equal(actual.sameSite, "Strict");
+      });
+    });
+
+    it('can add same site cookie property to `Lax`', async function() {
+      let cookie = createSameSiteCookieSpec('Lax');
+      let childUrl = fileserver.whereIs('child/childPage.html');
+      await driver.get(childUrl);
+      await driver.manage().addCookie(cookie);
+      await driver.manage().getCookie(cookie.name).then(function(actual) {
```

Comment:
The callback is not necessary since you are using async/await

```js
const actual = await driver.manage().getCookie(cookie.name);
assert.equal(actual.sameSite, "Strict");
```

Reply:
Replaced callback with async/await as suggested